index.bs

```diff
@@ -9791,17 +9810,29 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
       1. Append |navigable| to |navigables|.
 
+1. Otherwise, if |command parameters| [=map/contains=] <code>userContexts</code>:
+
+  1. Set |user contexts| to [=set/create|create a set=] with |command parameters|["<code>userContexts</code>"].
+
+  1. For each |user context id| of |user contexts|:
+
+    1. Set |user context| to [=get user context=] with |user context id|.
+
+    1. If |user context| is null, return [=error=] with [=error code=] [=no such user context=].
```

Comment:
nit: maybe iterate through the `|command parameters|["<code>userContexts</code>"]`, assert valid values and append them to existing map `|user contexts|` to increase readability and align with `contexts` logic one step above?